server/test/integration/dbTest/beginTransactionTest.js

```diff
@@ -108,5 +111,9 @@ describe('db.js integration tests: beginTransaction', () => {
     it('should return null', () => {
       expect(pgClient).to.be.null
     })
+
+    it('should try to rollback the transaction', () => {
+      expect(rollbackTransactionStub).to.be.called
```

Comment:
Cool. What do you think about adding `expect(rollbackTransactionStub).to.not.be.called` to the other test cases in this file?

Reply:
Ahh very this is a very good point - will do!